certbot-apache/certbot_apache/configurator.py

```diff
@@ -269,8 +293,13 @@ def _check_aug_version(self):
 
     def get_parser(self):
         """Initializes the ApacheParser"""
+        # If user provided vhost_root value in command line, use it
+        if self.option("vhost_root") != self.OS_DEFAULTS["vhost_root"]:
```

Comment:
All of the default `vhost-root` paths are parsed (with the exception of Ubuntu/Debian sites-available/sites-enabled scheme)  by the root Apache configuration. The only situation where this would be causing issues, would be if user has modified Apache root configuration to explicitly not parse the default VirtualHost path, but still wanting to create the VirtualHosts to that directory path. I would be surprised if stumbling upon such scenario. Then again, the experience has shown that there's a lot ot crazy Apache configurations out there in the wild.

Reply:
Yeah. What do you think about the suggestion of switching back to `self.conf` for the vhost root to preserve behavior here until this is fixed?